Build/linq2db.Source.props

```diff
@@ -22,12 +22,13 @@
 	</PropertyGroup>
 
 	<PropertyGroup>
-		<TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+		<TargetFrameworks>net45;net46;net47;net48;netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
I think I pulled this from linq2db: https://github.com/linq2db/linq2db/blob/2cd6eeb6984c652eafea352d26905b49b9dc31cd/Source/LinqToDB/LinqToDB.csproj#L9.

I can simplify it some but I do need to target net5.0 specifically. There is a difference in the package requirements between CORE 3.1 and .NET 5.0: https://github.com/devinlyons/Linq2DB4iSeries/blob/366077ead93a5d3efd40a7a0c9f0373a1efcc566/Build/linq2db.Tests.Providers.props#L26

Reply:
I see. Linq2db have more dependencies than we do so they tarther more framework versions. I still don't see why we need to target net5. Net472 however will need to be enevtually targeted because they added native support for async enumerables via a nuget package on 3.3.0.

Reply:
Fair enough. I will remove .NET 5.0. I'll be sure to add the net472 target when I finish the 3.3.0 update.